### DISCUSSION TITLE
Sync up battery health charging between Asus and G-helper

### DISCUSSION THREAD

**Discussion by thduc87** [seerge/g-helper#366] [Ideas]
I'm just wondering is it possible to sync up battery health charging between asus and g-helper so that when i change the setting in ghelper it will change in asus as well and vice versa. Thanks in advance.

**Comment by seerge** [comment]
@thduc87 check build that i have posted in https://github.com/seerge/g-helper/discussions/361, it literally does that already :)

**Reply by thduc87** [reply]
thanks, i already updated to latest build but when i change setting in myasus it does not reflect in g-helper and vice versa.

**Reply by seerge** [reply]
Myasus won’t cause their ui has 80% as only option.

But idea is that asus optimization service won’t overwrite custom values anymore

**Reply by thduc87** [reply]
ah i see, so we just disregard what myasus show? and ghelper did overwrite the value in customization.ini.

**Reply by seerge** [reply]
@thduc87 yes, every time g-helper sets “real” charge limit in bios, it will also put same value to ini. Mainly to fight accidental overwrites from asus services.

Reading value from ini makes no sense as it always will be 80 or 100 after setting it in myasus

**Reply by thduc87** [reply]
thanks for clarification, much appriciated.
